rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/azure/AzureBakeHandlerSpec.groovy

```diff
@@ -145,7 +145,7 @@ class AzureBakeHandlerSpec extends Specification implements TestDefaults{
       AzureBakeHandler azureBakeHandler = new AzureBakeHandler(azureBakeryDefaults: azureBakeryDefaults)
 
     when:
-      String templateFileName = azureBakeHandler.getTemplateFileName()
+      String templateFileName = azureBakeHandler.getTemplateFileName(azureBakeHandler.bakeOptions.baseImages[0])
```

Comment:
Would be nice if the `baseImages` had one that specified a `templateFile` so this test can be made to prove that the new override is respected.